chia/rpc/wallet_rpc_api.py

```diff
@@ -824,13 +826,16 @@ async def cat_get_asset_id(self, request):
         asset_id: str = wallet.get_asset_id()
         return {"asset_id": asset_id, "wallet_id": wallet_id}
 
-    async def get_offer_summary(self, request):
+    async def cat_asset_id_to_name(self, request):
```

Comment:
This diff looks weird because I merged it weird but the function is added back immediately below.